test-app/build.gradle

```diff
@@ -1,4 +1,3 @@
-// TODO: Move to the plugins{} block when AGP 4.2 stable is released
```

Comment:
I think I left this comment because AGP 4.2 is the first version that publishes the plugin markers necessary to use the `plugins` block, but the AGP documentation has not been updated to recommend using it yet, so I'd rather hold off until it becomes more standard